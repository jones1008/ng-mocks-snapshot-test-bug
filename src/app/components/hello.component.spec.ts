import {HelloComponent} from "./hello.component";
import {AppModule} from "../app.module";
import {MockBuilder, MockRender} from "ng-mocks";

describe('HelloComponent', () => {
  beforeEach(() => {
    return MockBuilder(HelloComponent, AppModule);
  });

  it('should render component correctly with Joe', () => {
    const fixture = MockRender(HelloComponent, {name: 'Joe'});
    expect(fixture).toMatchSnapshot()
  });

  it('should render component correctly with other name', () => {
    const fixture = MockRender(HelloComponent, {name: 'Jane'});
    expect(fixture).toMatchSnapshot()
  });
});
