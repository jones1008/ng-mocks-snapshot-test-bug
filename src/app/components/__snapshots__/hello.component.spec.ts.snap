// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloComponent should render component correctly with Joe 1`] = `
<mock-render
  __ngContext__="0"
>
  <hello>
    <h1>
      Hello Joe!
    </h1>
  </hello>
</mock-render>
`;

exports[`HelloComponent should render component correctly with other name 1`] = `
<mock-render
  __ngContext__={[Function Number]}
>
  <hello>
    <h1>
      Hello Jane!
    </h1>
  </hello>
</mock-render>
`;
